docs/getting_started/working_in_github.md

```diff
@@ -0,0 +1,48 @@
+# Getting the code
+
+-   Create your own Fork (once) (in web ui)
```

Comment:
Hmm, not sure but reading this it just seems strange. maybe something like this:
```suggestion
-   Create your own Fork (once), this can be done using the web ui.
```

Reply:
Removed this file.